lib/services/awsAdapter.js

```diff
@@ -44,6 +43,28 @@ function AWSAdapter (ec2) {
 			return result;
 		});
 	});
+
+	this.getPublicIPAddress = Bluebird.method(function (instanceID) {
```

Comment:
I believe that the use of `Bluebird.method()` here is unnecessary since the method already does all of it's work in a promise chain.